custom_components/easee/services.yaml

```diff
@@ -159,7 +159,7 @@ set_charger_dynamic_current:
       example: 10
 
 set_charger_max_current:
-  description: Set charger max current
+  description: Set charger max current OBS! Should not be used by automation (may wear out flash memory if set too frequently).
```

Comment:
Change OBS! to N.B.

Reply:
See your point, I went with "Caution:" which perhaps is cleaner. Agree?